website/docs/quick-start/install-atmos.mdx

```diff
@@ -78,6 +78,20 @@ Atmos has native packages for macOS and every major Linux distribution. We also
     ```shell
     nix-env -iA nixpkgs.atmos
     ```
+
+    To get the latest version, you may need to update the Nix config to add "unstable" packages. 
+
+    For example, in `~/.config/nixpkgs/config.nix` add the following line to the file:
+
+    ```console
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
```

Comment:
I would say that you may need to update your nix config to support the `unstable` packages channel. I wouldn't make it prescriptive how...that suggestion above means that every package will now be loaded from unstable, which may cause undesired behavior. Some people may prefer stable, then only missing packages from stable are loaded from unstable. And the way you do this is based on how you configure nix (standard, flakes, etc.)

Reply:
To be fair, it does say

> to get the latest version, you may

If you don't care about the latest version, then these instructions are moot.